qcodes/instrument/visa.py

```diff
@@ -258,3 +259,21 @@ def snapshot_base(self, update: bool = True,
         snap['timeout'] = self.timeout.get()
 
         return snap
+
+    def wait_until_complete(self, check_after: float, sleep_time: float) -> None:
```

Comment:
shall `sleep_time` have a default value?

Reply:
Yeah, it seems this completely depends on experiment and/or the instrument. I did not want to set a default for this reason. But can be done, if you think that would be more proper?

Reply:
i trust your judgement on this. if really needed, it can be added later.